tests/ops/integration_tests/saas/test_mailchimp_task.py

```diff
@@ -87,40 +94,6 @@ async def test_mailchimp_access_request_task(
     # links
     assert v[f"{dataset_name}:member"][0]["email_address"] == mailchimp_identity_email
 
-    logs = (
```

Comment:
same as above comments on the override test. i guess it seems more likely to me now that these were causing some inconsistencies in the tests :)